mongodump/mongodump_test.go

```diff
@@ -332,6 +332,60 @@ func setUpTimeseries(dbName string, colName string) error {
 	return nil
 }
 
+func setupTimeseriesWithMixedSchema(dbName string, collName string) error {
+	sessionProvider, _, err := testutil.GetBareSessionProvider()
+	if err != nil {
+		return err
+	}
+
+	client, err := sessionProvider.GetSession()
+	if err != nil {
+		return err
+	}
+
+	if err := client.Database(dbName).Collection(collName).Drop(nil); err != nil {
+		return err
+	}
+
+	createCmd := bson.D{
+		{"create", collName},
+		{"timeseries", bson.D{
+			{"timeField", "t"},
+			{"metaField", "m"},
+		}},
+	}
+
+	createRes := sessionProvider.DB(dbName).RunCommand(nil, createCmd)
+	if createRes.Err() != nil {
+		return createRes.Err()
+	}
+
+	// SERVER-84531 was only backported to 7.3.
```

Comment:
It looks like backports were approved for v6.0 and v7.0. Can we create a ticket blocked on those backports to update these tests, and link a TODO here?